src/components/links/button-as-link/index.tsx

```diff
@@ -33,11 +23,17 @@ const ButtonAsLink: React.FC<ButtonAsLinkProps> = ({
   ...rest
 }: ButtonAsLinkProps) => {
   const href = rest.disabled ? '' : url
-  return (
-    <StyledLink inverted={inverted} size={size} {...rest} to={href}>
-      {children}
-    </StyledLink>
-  )
+
+  const Component = isExternalURL(href) ? S.ExternalLink : S.InternalLink
```

Comment:
![Screen Shot 2021-02-07 at 1 13 10 PM](https://user-images.githubusercontent.com/15198653/107147831-74a1e800-6950-11eb-9b53-8b7420ac5625.png)

Reply:
Je tento error i pro `import { TranslatedLink } from 'gatsby-plugin-translate-urls'`? Tento by měl být použit v `button-as-link`.

Reply:
ano i pro `TranslatedLink`